hunt/app/views/animals/index.html.erb

```diff
@@ -46,6 +47,12 @@
               <td><%=animal.name%></td>
               <td><%=animal.state%></td>
               <td><%=animal.category%></td>
+              <td>
+                <%= if animal.season_open.present? && animal.season_close.present?
```

Comment:
Oh Interesting, I haven't seen this syntax before. Im used to this syntax:
```
<% if condition %>
  # do some stuff here
<% end %>
```

Anyways, here I think we'll still want to show the date, even if only one is set. Ruby's safe navigation operator (`&.`) might come in handy here. It will prevent errors if one of the arguments is nil. So, you could do `animal.season_open&.strftime('%b %d %Y')` and it won't error even if season_open is null.